polus-feature-extraction-plugin/src/utility.py

```diff
@@ -0,0 +1,1035 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Dec 12 20:42:45 2019
+
+@author: nagarajanj2
+"""
+import javabridge as jutil
+import bioformats
+import os
+import glob
+import numpy as np
+import pandas as pd
+from skimage import measure
+from skimage.measure import label
+from scipy.stats import skew
+from scipy.stats import kurtosis as kurto
+from scipy.stats import mode as mod
+from scipy import stats
+from operator import itemgetter
+import math
+import bfio.bfio as bfio
+from bfio.bfio import BioReader
+import difflib
+
+#Create class to read .ome.tif files
+class ConvertImage(object):
```

Comment:
Add class and method commenting according to the Google Python commenting specification.

http://google.github.io/styleguide/pyguide.html